sql-select/twenty-films-of-06.sql

```diff
@@ -0,0 +1,6 @@
+SELECT "title",
+       "description",
+       "releaseYear"
+  FROM "films"
+ WHERE "releaseYear" = '2006'
```

Comment:
Only put single quotes around strings. This is a number.

Reply:
Fixed